pages/Documents/WebsiteMonitoring/VisitInformationAPI/visit-information.md

```diff
@@ -12,72 +12,76 @@ indicator: chat
+This API allows access to visitor data only when the session is active and exists in memory. After a maximum of 6 hours, the session will expire and be removed from memory, so a 404 response (internal code 12) will be returned.
 
-**OAuth**
+#### OAuth
 
-This API supports OAuth 1.0 authentication patterns and requires SSL protocol. Please see [this document's overview](rt-interactions-visit-information-overview.html) for more information on how to retrieve your API keys.
+This API supports OAuth 1.0 authentication patterns and requires SSL protocol. Please see [this document’s overview](rt-interactions-visit-information-overview.html) for more information on how to retrieve your API keys.
 
-**Path Parameters**
+#### Path parameters
 
-| Parameter | Description | Type | Required | Notes |
+| Parameter | Description | Type | Required |
 | :--- | :--- | :--- | :--- |
-| accountId | LP site ID | string | Required |  |
-| visitorId | Visitor ID | string | Required | |
+| accountId | LP site ID | String | Required |
+| visitorId | Visitor ID | String | Required |
 
-**Query Parameters**
+#### Query parameters
 
-| Name     | Description | Type | Required |
-| :------ | :------- | :-------- | :--------| :--- |
-| filter | Set to "agent"  | string | Optional |
-| v | API version  | string | Required |
+| Name | Description | Type | Required |
+| :--- | :--- | :--- | :--- |
+| filter | Set to "agent" | String | Optional |
+| v | API version | string | Required |
 | sid | Session ID | string | Required |
 
 {: .attn-note}
 The **visitorId** and the **sid** parameters referenced above are retrieved from the Web Tag. Each visitor receives from the Tag their own **visitorId** (also called **vid**) and **sid** (also called **Session ID**) which you can then retrieve and pass to this API. For more information on these events, please see the Tag documentation [here](/lp-tag-engagement-window.html).
-**This method of retrieving the parameters is relevant to monitored sessions only**. If you're trying to retrieve information on unmonitored sessions (for example, chat windows built by you using our APIs and not the default Conversational Cloud window), you'll need to use the [App Engagement API](/rt-interactions-app-engagement-overview.html) instead, specifically the Create Session method which returns both of these parameters.
 
-**Request Example**
+{: .attn-note}
+**This method of retrieving the parameters is relevant to monitored sessions only**. If you’re trying to retrieve information on unmonitored sessions (for example, chat windows built by you using our APIs and not the default Conversational Cloud window), you’ll need to use the [App Engagement API](/rt-interactions-app-engagement-overview.html) instead, specifically the Create Session method which returns both of these parameters.
```

Comment:
(for example, chat windows you built using our APIs and not the default Conversational Cloud window)